ux.symfony.com/templates/base.html.twig

```diff
@@ -14,6 +14,7 @@
         <link rel="icon" href="/favicon.svg" sizes="any" type="image/svg+xml">
         <link rel="apple-touch-icon" href="/apple-touch-icon.png">
         <link rel="manifest" href="/site.webmanifest">
+        <meta name="theme-color" content="#212529">
```

Comment:
It feels like we should, perhaps, by sharing some values (somehow) between `base.html.twig` and the config file. The `meta` description is another, and also the `site.webmanifest`